TDL-06-EC2-Cloudwatch-S3/terraform-code-ec2-cw-s3/scripts/install_apache_cwagent.sh

```diff
@@ -0,0 +1,31 @@
+
+#!/bin/bash
+# install apache on EC2 instance
+yum update -y
+yum install httpd -y
+service httpd start
+chkconfig httpd on
+echo "<html><body>Test Cloudwatch Agent on Instance: $(hostname -f)</body></html>"  > /var/www/html/index.html
+
+# install AWS CLI
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+./aws/install
+
+# install the agent
+wget https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm 
+rpm -U ./amazon-cloudwatch-agent.rpm
+
+# create some missing files
+mkdir -p /usr/share/collectd
+touch /usr/share/collectd/types.db
```

Comment:
Do we need collectd? I did not notice there is collectd configured in config.json.
Further reading of collectd:

https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-custom-metrics-collectd.html